wagtail/images/image_operations.py

```diff
@@ -29,8 +29,97 @@ def construct(self, *args):
 # Transforms
 
 
+class ImageTransform:
+    """
+    Tracks transformations that are performed on an image.
+
+    This allows multiple transforms to be processed in a single operation and also
+    provides a matrix that can be used to transform the focal point of the image.
```

Comment:
Where is the matrix here?

Reply:
Oops, I removed the matrix (reasoning in PR description)